Misc/NEWS.d/next/Library/2025-09-03-18-26-07.gh-issue-138425.cVE9Ho.rst

```diff
@@ -0,0 +1,2 @@
+Partially evaluate undefined global generics in
```

Comment:
Mention of "generics" is imprecise; annotationlib doesn't know anything about generics and the issue affects any names that come from the globals.

Reply:
It could also be a union `undefined | int`, or an `Annotated` metadata, or just something that isn't a valid type annotation.

Maybe something like "Fix partial evaluation of ForwardRef objects that rely on names defined as globals."

Reply:
Perfect, thanks!